source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -0,0 +1,104 @@
+import AdaptiveCards_bridge
+import AppKit
+
+class ACRTextField: NSTextField {
+    var isDarkMode: Bool = false
+    
+    init(frame frameRect: NSRect, config: RenderConfig) {
+        super.init(frame: frameRect)
+        initialise(config: config)
+        setupConstraints(config: config)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func initialise(config: RenderConfig) {
+        self.isDarkMode = config.isDarkMode
```

Comment:
```suggestion
    private func initialise() {
        self.isDarkMode = config.isDarkMode
```
`private`
param can be ignored if config is stored as class property.